flow360/component/flow360_params/solvers.py

```diff
@@ -289,6 +297,7 @@ class TurbulenceModelSolverSST(TurbulenceModelSolver):
     """:class:`TurbulenceModelSolverSST` class"""
 
     model_type: Literal["kOmegaSST"] = pd.Field("kOmegaSST", alias="modelType", const=True)
+    model_constants: Optional[TurbulenceModelConstantsSST] = pd.Field(alias="modelConstants")
```

Comment:
every field present as an allowed type in Flow360Params structure, will be present in webUI available option. Therefore I think the best would be to add defaults to the `TurbulenceModelConstantsSA` and `TurbulenceModelConstantsSST`

Reply:
yes, added.